.gitattributes

```diff
@@ -4,8 +4,8 @@
 
 # Explicitly declare text files you want to always be normalized and converted
 # to native line endings on checkout.
-*.rs text eof=lf
-*.toml text eof=lf
+*.rs text eol=lf
+*.toml text eol=lf
 *.frag text
 *.vert text
```

Comment:
Shouldn't the `*.frag text` and `*.vert text` rules be replaced by a rule for `*.wgsl text`?

Reply:
Assuming we want to have:
```
*.frag text eol=lf
*.vert text eol=lf
*.wgsl text eol=lf
```
in order to keep supporting both the old and the new renderer until the old one gets removed?

Reply:
It wouldn't hurt to have rules for all three.